Tasks/Common/VstsAzureHelpers_/InitializeAzModuleFunctions.ps1

```diff
@@ -132,16 +132,9 @@ function Initialize-AzSubscription {
         }
 
     } elseif ($Endpoint.Auth.Scheme -eq 'ManagedServiceIdentity') {
-        $accountId = $env:BUILD_BUILDID 
-        if($env:RELEASE_RELEASEID){
-            $accountId = $env:RELEASE_RELEASEID 
-        }
-        $date = Get-Date -Format o
-        $accountId = -join($accountId, "-", $date)
-        $access_token = Get-MsiAccessToken $Endpoint
```

Comment:
Can we remove this method?

Reply:
No, it is being used by Azure and AzureRm modules as the code is under Common